src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -0,0 +1,133 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+namespace GeneralizedHarmonic {
+template <size_t Dim>
+struct SpacetimeMetric : db::DataBoxTag {
+  using type = tnsr::aa<DataVector, Dim, Frame::Grid>;
+  static constexpr db::DataBoxString_t label = "SpacetimeMetric";
+};
+
+/*!
+ * \brief Auxiliary variable which is combination of time and space derivatives
+ * of the spacetime metric
+ *
```

Comment:
Instead of 'a combination of time and space derivatives', say that  \f$\Pi_{ab} = -n^c\partial_c \psi_{ab}\f$ where \f$n^c\f$ is the timelike unit normal to the slice.

Reply:
I typically like being precise, but isn't putting a formula in a `brief` overdoing it? I would favor a phrase like, `the derivative of the spacetime metric in the timelike direction normal to the slice` or similar.

Reply:
Or we could say it's a conjugate momentum, and have the formula in the details section. Reading this again, I don't like the "combination of time and space derivatives" phrasing